### PR TITLE
update gateway docs

### PR DESCRIPTION
**Pull request by pwright**

### CODE REVIEW

File: modules/cli/pages/gateway.adoc [hunk 1]
```diff
@@ -0,0 +1,352 @@
...
+The following are valid entries in a gateway definition YAML file.
+
+name:: Name of gateway
+bindings.name:: Name of binding for a single host.
+bindings.host:: Hostname of local service.
+bindings.service:: Definition of service you want to be available on service network.
+bindings.service.address:: Address on the service network, name and port.
+bindings.service.protocol:: Skupper protocol, `tcp`, `http` or `http2`.
+bindings.service.ports:: A single port that becomes available on the service network. 
+bindings.service.exposeIngress:: (optional) The ingress mode for the service.
```
Comment (pwright) [inline modules/cli/pages/gateway.adoc:330]: @ajssmith What does the exposeIngress mode do?
You can specify it in a gateway yaml file, but I'm not sure what values are valid?

File: modules/cli/pages/gateway.adoc [hunk 2]
```diff
@@ -0,0 +1,352 @@
...
+bindings.service.tlscredentials:: (optional) The TLS certificate and key for the service.
+bindings.service.tlscertauthority:: (optional) The TLS public certificate.
+bindings.service.publishnotreadyaddresses:: Specify whether to sync services relating to Kubernetes objects scheduled but not yet available.
```
Comment (pwright) [inline modules/cli/pages/gateway.adoc:333]: @ajssmith Is this ok?

btw just the lines between my comments need to be reviewed

File: modules/cli/pages/gateway.adoc (hunk 1 above)
Comment (ajssmith) [reply modules/cli/pages/gateway.adoc:330]: The gateway has (yet another) convention where ingress/egress conveyed direction of service connectivity with respect to application network as gateway is on the edge. TL/DR, ingress == listener, egress == connector wrt router configuration.

File: modules/cli/pages/gateway.adoc (hunk 2 above)
Comment (ajssmith) [reply modules/cli/pages/gateway.adoc:333]: Suggest that publishnotreadyaddresses be omitted as gateway does not participate in service sync

**Review by ajssmith — APPROVED** [review]
lgtm,  thnx

**Pull request merged**